js/utility/errorMessage.js

```diff
@@ -0,0 +1,7 @@
+export default function createErrorMessage(){
+const errorMessage = document.createElement('div');
+errorMessage.textContent = 'Please add task description.';
+errorMessage.style.color = 'red';
+errorMessage.id = 'error-message';
+return errorMessage;
+}
```

Comment:
Function name should be concise about what they do. In this instance this function creates an error message element and returns it. 

```suggestion
export default function createErrorMessageElement(){
const element = document.createElement('div');
element.textContent = 'Please add task description.';
element.style.color = 'red';
element.id = 'error-message';
return element;
}
```

Reply:
I've changed it to errorMessageElement.